### DISCUSSION TITLE
Struggling with Logger.xml

### DISCUSSION THREAD

**Discussion by bchah** [OvenMediaLabs/OvenMediaEngine#1479] [Help] [answered]
It feels like I am doing something wrong in Logger.xml. OME is not printing the logs as desired. I want critical log level only for some, info log level for others, and warn log level for everything else. Have I done this in the wrong order? In particular, HTTP.Server is quite verbose. I tried escaping the dot (\.) and trying a regex like "HTTP.*" but it still prints all HTTP.Server errors.

```
<Logger version="2">

	<!-- Log file location -->
	<Path>/log/engine</Path>
	<!-- Disable some logs -->
	<Tag name="SRT" level="critical" />
	<Tag name="HTTP.Server" level="critical" />
	<Tag name="FFmpeg" level="critical" />
	<Tag name="APIController" level="critical" />
	<!-- Log level: [debug, info, warn, error, critical] -->
	<Tag name="OvenMediaEngine" level="info" />
	<Tag name="Monitor" level="info" />
	<Tag name="TranscodeStream" level="info" />
	<Tag name=".*" level="warn" />
</Logger>
```

**Comment by dimiden** [comment, marked as answer]
@bchah 
To filter logs starting with ```HTTP.```, you should specify exactly ```name="HTTP\..*"```, but regardless of that, both ```name="HTTP.Server"``` and ```name="HTTP."``` should be filtered as ```HTTP.Server``` logs.
There is a high possibility that ```Logger.xml``` is not loaded correctly; do other logs get filtered?
